src/executor/executor.c

```diff
@@ -0,0 +1,252 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <getopt.h>
+#include <unistd.h>
+
+#include "alloc-util.h"
+#include "build.h"
+#include "execute-serialize.h"
+#include "exit-status.h"
+#include "fdset.h"
+#include "fd-util.h"
+#include "fileio.h"
+#include "getopt-defs.h"
+#include "parse-util.h"
+#include "pretty-print.h"
+#include "sd-messages.h"
+#include "unit-serialize.h"
+
+static FILE* arg_serialization = NULL;
+
+static int help(void) {
+        _cleanup_free_ char *link = NULL;
+        int r;
+
+        r = terminal_urlify_man("systemd", "1", &link);
+        if (r < 0)
+                return log_oom();
+
+        printf("%s\n\n"
+               "Sandbox and execute processes.\n\n"
+               "  -h --help              Show this help and exit\n"
+               "  --version              Print version string and exit\n"
```

Comment:
we usullay give short options a column of its own.

btw, for new code, please generate the output with the highlighted description.

see how src/notify/notify.c does it for example, as a good and simple example.